doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -47,25 +47,24 @@ According to CNCF TOC (Technical Oversight Committee), following is the definiti
 
 >“Cloud native technologies empower organizations to build and run **scalable** applications in modern, **dynamic environments** such as public, private, and hybrid clouds. Containers, **service meshes**, **microservices**, **immutable infrastructure**, and **declarative APIs** exemplify this approach.
 
->These techniques enable **loosely coupled** systems that are **resilient**, **manageable**, and **observable**. Combined with **robust automation**, they allow engineers to make high-impact changes frequently and predictably with minimal toil.
+>These techniques enable **loosely coupled** systems that are **resilient**, **manageable**, and **observable**. Combined with **robust automation**, they allow engineers to make **high-impact changes frequently and predictably** with minimal toil.
 
 >The Cloud Native Computing Foundation seeks to drive adoption of this paradigm by fostering and sustaining an ecosystem of open source, vendor-neutral projects. We democratize state-of-the-art patterns to make these innovations accessible for everyone”
 
-The definition above is very application centric and look at cloud native from the application point of view, in here, we will try and relate those definitions to the infrastructure on which cloud native applications run. For the purposes of this document, a Cloud Native Network Function (CNF) is a type of cloud native application.
-
-- **scalable**:
-- **dynamic environments**:
-- **service meshes**:
-- **microservices**:
-- **immutable infrastructure**:
-- **declarative APIs**:
-- **loosely coupled**:
-- **resilient**:
-- **manageable**:
-- **observable**:
-- **robust automation**:
-
-> The CNCF TUG is also working on a set of Cloud Native Principles at more detail than the existing CNCF definition: [Expanded Cloud Native Principles](https://github.com/vulk/cloud-native-principles/blob/master/cloud-native-principles.md).  Many of the principles above are taken from this document.
+The CNCF TUG (Telecom User Group) is also working on a set of Cloud Native Principles at more detail than the existing CNCF definition: [Expanded Cloud Native Principles](https://github.com/vulk/cloud-native-principles/blob/master/cloud-native-principles.md). There are many similarities, but the key principles from both, which are applicable to this document, are:
+
+- **scalable**
+- **dynamic environments**
+- **service meshes**
+- **microservices**
+- **immutable infrastructure**
+- **declarative APIs**
+- **loosely coupled**
+- **resilient**
+- **manageable**
+- **observable**
+- **robust automation**
+- **high-impact changes frequently and predictably**
```

Comment:
"robust automation" and "high impact changes" are not in the list of cloud native principles (at the given link), but just in the endnotes as part of reasoning. I have to say I don't know what is "robust automation" and I would perfer to not do high impact changes.....

Reply:
The issue this PR relates to is specifically to add "high impact changes" in :)

I would argue the ability to make high impact changes is one of the many business benefits of "cloud native" and robust automation is one concept to help achieve that by ensuring consistency of environments and software.

Reply:
@tomkivlin I understand what is the intent here -- we are asking for "high volume changes (to the infrastructure) w/o impact" to workloads. Maybe we could make that clear.

Should the list be ordered with, say, microservices, immutable infrastructure and declarative APIs on the top?

Should "scalable" be qualified -- "horizontal" or "outward"?

Reply:
> @tomkivlin I understand what is the intent here -- we are asking for "high volume changes (to the infrastructure) w/o impact" to workloads. Maybe we could make that clear.

@pgoyal01 given the wording is a direct reference to the CNCF cloud native definition, I'd rather not change it.  However, we can (in a different PR) start to flesh out each list item with our meaning/take on it as you've articulated - I like your wording.





> Should the list be ordered with, say, microservices, immutable infrastructure and declarative APIs on the top?

At the moment it's ordered as they appear in the definition, I'd rather not start to put order of importance at this stage unless there is clear consensus as to what that order it.



> Should "scalable" be qualified -- "horizontal" or "outward"?

As per first point, I'd rather qualify in a separate PR please.